pkg/operator/credentialsrequest/credentialsrequest_controller.go

```diff
@@ -336,6 +354,23 @@ func isAdminCredSecret(namespace, secretName string) bool {
 	return false
 }
 
+// isAWSResourceTagsPresent returns true if the AWS resourceTags are present in the Infrastructure resource
+func isAWSResourceTagsPresent(event client.Object) bool {
```

Comment:
Just a personal preference but `hasResourceTags` sounds better to me.

Reply:
Updated